packages/components/src/placeholder/index.js

```diff
@@ -46,8 +46,8 @@ function Placeholder( {
 	let modifierClassNames;
 	if ( typeof width === 'number' ) {
 		modifierClassNames = {
-			'is-large': width >= 320,
-			'is-medium': width >= 160 && width < 320,
+			'is-large': width >= 480,
```

Comment:
320 is not "large", and the container query system is less useful as a result. 

This is nevertheless a change that makes it worth testing other containers that might be targetting the `is-large` class.